sdk/python/cmd/pulumi-language-python/testdata/classes/projects/l1-config-types/__main__.py

```diff
@@ -0,0 +1,16 @@
+import pulumi
+
+config = pulumi.Config()
+a_number = config.require_float("aNumber")
+pulumi.export("theNumber", a_number + 1.25)
+a_string = config.require("aString")
+pulumi.export("theString", a_string + " World")
+a_map = config.require_object("aMap")
+pulumi.export("theMap", {
+    "a": a_map["a"] + 1,
+    "b": a_map["b"] + 1,
+})
+an_object = config.require_object("anObject")
+pulumi.export("theObject", an_object["prop"][0])
```

Comment:
Given the reason we're writing this, this is a bit annoying, right? As in, we were hoping this would yield an expression of the form `an_object.prop[0]`?

Reply:
Yeh but I think this is the best we can do in python. It works at least we can revisit it later.